### ISSUE TITLE
can not auto build docker images

### ISSUE THREAD

**Issue by zh794390558** [didi/delta#119]
```
Building in Docker Cloud's infrastructure...
Cloning into '.'...
Warning: Permanently added the RSA host key for IP address '140.82.113.4' to the list of known hosts.
fatal: no submodule mapping found in .gitmodules for path 'egs/conll2003/pretrain/v1/local/bilm'
please ensure the correct public key is added to the list of trusted keys for this repository and the remote branch exists. (128)
```

refs:
https://docs.docker.com/docker-hub/builds/link-source/
https://github.com/docker/hub-feedback/issues/416
https://github.com/docker/hub-feedback/issues/213
https://github.com/docker/hub-feedback/issues/633

**Comment by zh794390558** [comment]
goto https://cloud.docker.com/repository/docker/zh794390558/delta/builds/edit and push `Generate` button in deploy key.

**Comment by zh794390558** [comment]
`fatal: no submodule mapping found in .gitmodules for path 'egs/conll2003/pretrain/v1/local/bilm'`

**Comment by zh794390558** [comment]
fix by #120
